Source/Core/DiscIO/CompressedBlob.cpp

```diff
@@ -133,7 +134,7 @@ bool CompressedBlobReader::GetBlock(u64 block_num, u8* out_ptr)
     z.avail_in = comp_block_size;
     if (z.avail_in > m_header.block_size)
     {
-      PanicAlertFmt("We have a problem");
+      ERROR_LOG_FMT(DISCIO, "We have a problem");
```

Comment:
Can we make this log entry more useful while we're at it? As it is now, it doesn't help anyone (other than letting us know we hit this line)

Reply:
I've replaced it with a message that I *think* is correct.